### PR TITLE
Did rpc

### PR DESCRIPTION
**Pull request by osinakayah**

### CODE REVIEW

**Review by maycon-mello — APPROVED** [review]

File: packages/core/lib/services/dids/service-rpc.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import {RpcService} from '../rpc-service-client';
+import {serviceName} from './config';
+
+export class DIDServiceRPC extends RpcService {
+  constructor() {
+    super(serviceName);
+  }
+  keypairToDidKeyDocument(params) {
```
Comment (maycon-mello) [inline packages/core/lib/services/dids/service-rpc.js:8]: Can we define some types for those parameters?

Comment (maycon-mello) [reply packages/core/lib/services/dids/service-rpc.js:8]: And maybe add validation

Comment (maycon-mello) [reply packages/core/lib/services/dids/service-rpc.js:8]: Similar to other rpc services we have

File: packages/core/lib/services/dids/index.test.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      correlation: ['4058a72a-9523-11ea-bb37-0242ac130002'],
+      controller: 'did:key:z6MkjjCpsoQrwnEmqHzLdxWowXk5gjbwor4urC1RPDmGeV8r',
+      type: 'Ed25519VerificationKey2018',
+      privateKeyBase58:
+        '3CQCBKF3Mf1tU5q1FLpHpbxYrNYxLiZk4adDtfyPEfc39Wk6gsTb2qoc1ZtpqzJYdM1rG4gpaD3ZVKdkiDrkLF1p',
+      publicKeyBase58: '6GwnHZARcEkJio9dxPYy6SC5sAL6PxpZAB6VYwoFjGMU',
+    };
+    const res = await service.keypairToDidKeyDocument({keypairDoc});
+    expect(DIDKeyManager.keypairToDidKeyDocument).toBeCalledWith(keypairDoc);
+    expect(res).toBeDefined();
```
Comment (mike-parkhill) [inline packages/core/lib/services/dids/index.test.js:28]: can we assert anything specific about `res`?

File: packages/core/lib/services/dids/index.test.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        '#z6Mks8mvCnVx4HQcoq7ZwvpTbMnoRGudHSiEpXhMf6VW8XMg',
+      ],
+      capabilityDelegation: [
+        '#z6Mks8mvCnVx4HQcoq7ZwvpTbMnoRGudHSiEpXhMf6VW8XMg',
+      ],
+      keyAgreement: ['#z6LScrLMVd9jvbphPeQkGffSeB99EWSYqAnMg8rGiHCgz5ha'],
+    };
+    const res = await service.getDidResolution({didDocument});
+    expect(DIDKeyManager.getDidResolution).toBeCalledWith(didDocument);
+    expect(res).toBeDefined();
```
Comment (mike-parkhill) [inline packages/core/lib/services/dids/index.test.js:77]: same here

File: packages/core/lib/services/dids/index.test.js (hunk 2 above)
Comment (SamHellawell) [reply packages/core/lib/services/dids/index.test.js:28]: yes, we should check that the DID ID is correct relative to the public key (its fingerprint)

**Pull request merged**